src/pages/components/tabs/style.mdx

```diff
@@ -33,17 +174,102 @@ Tab label should be set in sentence case, and should not exceed three words.
 
 ## Structure
 
-| Class                 | Property                    | px / rem | Spacing token |
-| --------------------- | --------------------------- | -------- | ------------- |
-| `.bx--tabs__nav-item` | border-bottom               | 2px      | –             |
-| `.bx--tabs__nav-link` | padding-left, padding-right | 16 / 1   | `$spacing-05` |
-| `.bx--tabs__nav-link` | padding-top, padding-bottom | 8 / 0.5  | `$spacing-03` |
-| `bx--tabs__nav-item`  | margin-left                 | 1px      | -             |
+### Line tabs
+
+| Element         | Property                    | px / rem   | Spacing token |
+| --------------- | --------------------------- | ---------- | ------------- |
+| Tab             | height                      | 40 / 2.5   | –             |
```

Comment:
Its hard to tell in markdown but looking at the preview some of your dashes are different length. They all should be using the en dash —

Reply:
made sure all were en dashes!